homeassistant/components/notify/html5.py

```diff
@@ -131,29 +140,79 @@ def post(self, request):
         return self.json_message("Push notification subscriber registered.")
 
 
+class HTML5PushCallbackView(HomeAssistantView):
```

Comment:
I did a platform specific implementation because html5 and iOS are different in that one has authentication information easily accessible (iOS) but html5 does not, meaning html5 is the only one needing callback URLs (and in fact, it would degrade the iOS experience to use the same URL system). iOS also has support for many more capabilities in push notifications actions then HTML5 does, such as accepting text input and coloring buttons etc.